LondonBritishLibrary/orient/BLorient7946.xml

```diff
@@ -0,0 +1,182 @@
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="124r"/>
+                            <title ref="NAR0184ForeignWords">Amharic-English Vocabulary</title>
+                            <note>The transcription (without gemination) of the Amharic words is given systematically as far as <locus target="#84v"/>.
+                                Vocables are illustrated with examples. The total number of vocables is 5632 (cf. <locus target="#2v"/> with a list of letters
+                                giving the number of entries corresponding to each of them, similar to <ref target="#a1"/>).</note>
+                            <textLang xml:lang="en" otherLangs="am"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus target="#124v"/>
+                            <title ref="NAR0184ForeignWords">Beginning of an English-Amharic Vocabulary</title>
+                            <explicit xml:lang="en">Admire, v.a. <foreign xml:lang="am">አደነቀ፡ አከበረ፡</foreign></explicit>
+                            <textLang xml:lang="en" otherLangs="am"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="125r" to="126r"/>
+                            <title ref="NAR0184ForeignWords">Amharic Alphabet</title>
+                            <note>The seven orders of the Amharic syllabary are arranged as follows: (I) First order, (II) fourth order, (III) fifth order,
+                                (IV) third order, (V) sixth order, (VI) second order, (VII) seventh order. Under each letter of the first order its 
+                                pronunciation and the corresponding page of the Vocabulary are given.</note>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus target="#126v"/>
+                            <title ref="NAR0184ForeignWords">Remarks on the Amharic script</title>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <locus target="#127r"/>
+                            <title ref="LIT7041LordsPrayerAmh">The Lord's prayer in Amharic</title>
+                            <note></note>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="paper"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">1+123+4</measure>
+                                    <measure type="blank" unit="page">3</measure><locus target="#1r #1v #127v"/>
```

Comment:
I wonder if this should be also "leaf". 1r, 1v 127v  are designations for folia. If you will use "page", it will probably visualized as pp (if at all).

Reply:
Yes, I will change it in a minute in a new branch.